Test_STAR_Edge_Cases.py

```diff
@@ -0,0 +1,52 @@
+import json
+import pandas as pd
+import numpy as np
+from STAR import STAR, TrueTie
+import pytest
+
+
+class TestSTAR:
+    def test_election_with_one_record_only(self):
+        columns = ['Allison', 'Bill', 'Carmen']
+        election = [[5, 2, 1, 4]]
+        ballots = pd.DataFrame(columns=columns, data=election)
+        results = STAR(ballots)
+        assert results['elected'] == ['Allison']
+
+    def test_mismatch_between_number_of_candidates_and_ballots(self):
```

Comment:
since STAR takes a DataFrame as input, this test feels a little strange to me since it creates its own error. Otherwise looks good!